index.html

```diff
@@ -3001,6 +2999,38 @@ <h3>
 
         </section>
 
+        <section class="normative">
+            <h3>
+                Add Index
+            </h3>
+
+            <p>
+  Encrypted Documents can have embedded key-value attribute pairs, as seen in <a href="#blinded-document-attributes"></a>.
```

Comment:
```suggestion
  Encrypted Documents can have embedded attribute key-value pairs, as seen in <a href="#blinded-document-attributes"></a>.
```

... or maybe "attribute name-value pairs"?

Reply:
Done!